index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BASKETBALL TEAM</title>
+</head>
+<body>
+    <header>
+<nav>
+<div> <img src="/images/Basketball logo.jpg" alt="Basketball team logo" width="90" height="auto">
+    <p>CEE TEAM</p>
+</div>
```

Comment:
![image](https://github.com/marcyl254/HTML-Images-and-Links/assets/83514256/6e9b6d52-e47d-419c-b07f-5326b501fb89)

#OPTIONAL

Kindly ensure all the images are rendering correctly for all users. The logo seems to be missing..
Kindly Update that. 💭